build.sh

```diff
@@ -19,42 +21,45 @@ time xcodebuild clean test \
     | xcpretty &&
 cat build.log | sh debug-time-function-bodies.sh &&
 rm -rf $DERIVED_DATA &&
+# Run test on tvOS
 time xcodebuild clean test \
     -project LayoutKit.xcodeproj \
     -scheme LayoutKit-tvOS \
-    -sdk appletvsimulator10.1 \
+    -sdk appletvsimulator10.2 \
     -derivedDataPath $DERIVED_DATA \
     -destination 'platform=tvOS Simulator,name=Apple TV 1080p,OS=9.2' \
-    -destination 'platform=tvOS Simulator,name=Apple TV 1080p,OS=10.0' \
+    -destination 'platform=tvOS Simulator,name=Apple TV 1080p,OS=10.2' \
     OTHER_SWIFT_FLAGS='-Xfrontend -debug-time-function-bodies' \
     | tee build.log \
     | xcpretty &&
 cat build.log | sh debug-time-function-bodies.sh &&
 rm -rf $DERIVED_DATA &&
+# Build sample app
 time xcodebuild clean build \
     -project LayoutKit.xcodeproj \
     -scheme LayoutKitSampleApp \
-    -sdk iphonesimulator10.2 \
+    -sdk iphonesimulator10.3 \
     -derivedDataPath $DERIVED_DATA \
     -destination 'platform=iOS Simulator,name=iPhone 6,OS=9.3' \
     -destination 'platform=iOS Simulator,name=iPhone 6 Plus,OS=9.3' \
-    -destination 'platform=iOS Simulator,name=iPhone 7,OS=10.2' \
-    -destination 'platform=iOS Simulator,name=iPhone 7 Plus,OS=10.2' \
+    -destination 'platform=iOS Simulator,name=iPhone 7,OS=10.3' \
+    -destination 'platform=iOS Simulator,name=iPhone 7 Plus,OS=10.3' \
     OTHER_SWIFT_FLAGS='-Xfrontend -debug-time-function-bodies' \
     | tee build.log \
     | xcpretty &&
 cat build.log | sh debug-time-function-bodies.sh &&
 rm -rf $DERIVED_DATA &&
+# Run test on iOS
```

Comment:
Thank you for adding these comments! Could you please add a blank line before each comment, just to make it even easier to scan.